<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintainWin</class>
 <widget class="QWidget" name="MaintainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MaintainWin</string>
  </property>
  <widget class="QPushButton" name="pbFlush">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Back Flush</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbEnhancedFlush">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Enhanced Flush</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClean">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Clean</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPrimeDiluent">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Prime Diluent</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPrimeLyse">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Prime Lyse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPark">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>140</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Park</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbWholeFill">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>140</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Whole Fill</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbHelp">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>350</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbInit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Init</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbEnhancedClean">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>70</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Enhanced Clean</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPulse">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Pulse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbTscalibrate">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Tscalibrate</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
